iothub/device/src/Transport/Amqp/AmqpUnit.cs

```diff
@@ -231,12 +231,12 @@ private async Task EnsureMessageReceivingLinkAsync(TimeSpan timeout, bool enable
                     {
```

Comment:
Seems like this method name should be `EnsureMessageReceivingLinkIsOpenAsync`

Reply:
This is fair, yes; I'll rename it.